experiments/gromacs/cuda/ramble.yaml

```diff
@@ -18,13 +18,11 @@ ramble:
             set:
               OMP_PROC_BIND: close
               OMP_PLACES: cores
-              OMP_NUM_THREADS: '{omp_num_threads}'
           variables:
             experiment_setup: ''
-            n_ranks: '{processes_per_node} * {n_nodes}'
-            processes_per_node: '2'
+            n_ranks_per_node: '2'
             n_nodes: '4'
-            omp_num_threads: '10'
+            n_threads_per_proc: '10'
```

Comment:
So, there is only 2 ways in current use to launch multiple threads on the CPUs:
- pthreads (super low level, rarely if ever used by the application - although the tools might use these)
- OpenMP threads, which is what is used here.  OMP_NUM_THREADS is an environment variable everyone would immediately recognized, and I think we can reasonably assume that this is the only type of threads people will use on HPC CPUs.  I wonder if it is worth it to stick with the convention here for familiarity, and keep the omp_num_threads name.

Reply:
I think users will need to look up the list of experiment variables they need to set either way; I figured when they read through this, that `n_threads_per_proc` would be clear. And as you say, my main intent was to not constrain the term to one threading model.

FWIW I don't think anything is stopping me from adding the possibility of using `omp_num_threads` as an alias of `n_threads_per_proc` i.e. allow both, if we want the user to be able to guess what variables they set.

Reply:
I've added `omp_num_threads` as an alias (and added a mechanism for alias processing, but if we decide we just want one name for everything, I can remove `n_threads_per_proc` and the alias logic).